src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -290,6 +292,10 @@ object DemuxFastqs {
       |To output with Illumina conventions that match `bcl2fastq` and `BCLconvert`, use:
       |
       |`--fastq-skip-read-numbers=true --fastq-include-sample-barcodes=true --illumina-file-names=true`
+      |
+      |To output reads with a passing filter flag, encoded as 0 (not control) or >0 (an internal control) in the <is filter> flag of the header comment, use:
+      |
+      |`--keep-only-passing=true`
```

Comment:
This sounds to me like it's mixing two things - the `passing filter` flag and the `is control` flag/code.  How about

```suggestion
      |By default all input reads are output.  If your input FASTQs contain reads that do not pass filter (as defined by the Y/N filter flag in the FASTQ comment) these can be filtered out during demultiplexing using the `--skip-failing-reads` option.
```